client/src/main.rs

```diff
@@ -154,6 +154,14 @@ pub struct Opt {
     /// How long to wait before the auto flush check fires
     #[clap(long, global = true)]
     flush_timeout: Option<u32>,
+
+    /// IP:Port for the oximeter register address
```

Comment:
Might be a nit, but this is really Nexus's address.

Reply:
Nit for nit question, is the address where we register with Oximeter always going to be Nexus (at least the current plan)?